Documentation/dev/developer-guide.md

```diff
@@ -147,4 +141,8 @@ helm template CHART_DIR -x PATH_TO_TEMPLATE/file.yaml
 ## Developer install
 
 Developers should generally use the [manual-install guide](../manual-install.md) as it offers the most flexibility when installing.
-If you need a minimal storage configuration with no external dependencies, use the [manifests/metering-config/hdfs-minimal.yaml](../../manifests/metering-config/hdfs-minimal.yaml) example configuration.
+If you need a minimal storage configuration with no external dependencies, use the [manifests/metering-config/hdfs-minimal.yaml](manifests/metering-config/hdfs-minimal.yaml) example configuration.
```

Comment:
That path is local to the repository so changing that path leads to a `Page not found` error from Github.

Reply:
Actually I pulled this change in accidentally, didn't make it explicitly. I'll change it back

Reply:
I was actually using Github's editor, it must have tweaked the URL